funcbench/README.md

```diff
@@ -32,7 +32,7 @@ Flags:
       --nocomment            Disable posting of comment using the GitHub API.
       --owner="prometheus"   A Github owner or organisation name.
       --repo="prometheus"    This is the repository name.
-      --github-pr=GITHUB-PR  GitHub PR number to pull changes from and to post
+      --github-pr=GITHUB-PR  GitHub PR number to pull changes from and to pos
```

Comment:
I don't know why `make docs` create a typo here, in the source it's `post`, can you take a look? @geekodour

Reply:
Could you try running `make build docs` might be some previous build in which the typo was there. I just tried running make docs on your PR, it's working as expected for me.

Reply:
I tried several times, result the same typo.
I updated README manually. Let's get this PR done first.